browser/resources/settings/brave_data_collection_page/brave_data_collection_page.html

```diff
@@ -7,7 +7,7 @@
       on-settings-boolean-control-change="onP3AEnabledChange_">
   </settings-toggle-button>
 </template>
-<template is="dom-if" if="[[!isStatsReportingDisabledByPolicy_]]">
+<template is="dom-if" if="[[!isStatsReportingEnabledManaged_]]">
```

Comment:
it would be great if this also has same behavior with https://github.com/brave/brave-core/pull/30594/files#diff-a147331458e847653c2f204f521eaa2e504e8c8c9e17cf8259d65ae6fe0733c9R109

Reply:
i agree, but unfortunately i don't think it's possible with local state prefs. the settings private API only supports profile prefs AFAIK